### DISCUSSION TITLE
Nested count bug.

### DISCUSSION THREAD

**Discussion by Limonische** [strapi/strapi#8836] [Legacy Topics]
So there is currently this issue #7631. But i need to make infinite loading functionality with deep filtering by categories asap. Does anyone know any workaround for this issue for REST?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
